src/main/java/com/google/api/codegen/transformer/InitCodeTransformer.java

```diff
@@ -605,6 +617,19 @@ private InitValueView getInitValue(MethodContext context, InitCodeNode item) {
       fieldSetting.required(
           fieldConfig != null
               && requiredFieldSimpleNames.contains(fieldConfig.getField().getSimpleName()));
+      String formatMethodName = "";
+      String transformParamFunctionName = "";
+      if (context.getFeatureConfig().useResourceNameConverters(fieldConfig)) {
+        if (fieldConfig.getField().isRepeated()) {
+          // TODO support repeated one-ofs
```

Comment:
Should we create an issue to track this?

Reply:
https://github.com/googleapis/toolkit/issues/1806